upgrade/satellite.py

```diff
@@ -65,6 +66,8 @@ def satellite6_setup(os_version):
         # start's/enables/install's ntp
         # Check that hostname and localhost resolve correctly
         execute(install_prerequisites, host=sat_host)
+        # setup RHEV certificate so it can be added as a CR
+        execute(setup_rhv_ca, host=sat_host)
```

Comment:
Instead of here (i.e `satellite6_setup`), add this task as a part of : https://github.com/SatelliteQE/satellite6-upgrade/blob/079e2086107d00076beecfb860a72e87ac47bb0e/upgrade/helpers/tasks.py#L575

This func is specifically designed to run post-upgrade-tasks to prepare prerequisites for upgrade tests.